README.md

```diff
@@ -1 +1,67 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+This project is to set up github supository and set up linters on a portfolio project.
+The project is in a nub=mber of sections wich inclusde the headers, the name and title, the introduction and my abilities sections
+
```

Comment:
- [OPTIONAL] Please try to be more clear about what is your project in it's description and double-check the typos.